src/test/glob.test.ts

```diff
@@ -76,29 +80,60 @@ test.before.each(async (ctx) => {
         }
       }
 
-      let actual, error;
-      try {
-        actual = await glob(patterns, {
-          cwd: rig.resolve(cwd),
-          followSymlinks,
-          includeDirectories,
-          expandDirectories,
-          throwIfOutsideCwd,
-        });
-      } catch (e) {
-        error = e;
-      }
-      if (expected === 'ERROR') {
-        if (error === undefined) {
-          assert.unreachable('Expected an error');
+      // One-time globbing (via fast-glob).
+      if (mode === 'once') {
+        let actual, error;
+        try {
+          actual = await glob(patterns, {
+            cwd: rig.resolve(cwd),
+            followSymlinks,
+            includeDirectories,
+            expandDirectories,
+            throwIfOutsideCwd,
+          });
+        } catch (e) {
+          error = e;
+        }
+        if (expected === 'ERROR') {
+          if (error === undefined) {
+            assert.unreachable('Expected an error');
+          }
+        } else if (error !== undefined) {
+          throw error;
+        } else if (actual === undefined) {
+          throw new Error('Actual was undefined');
+        } else {
+          const actualPaths = actual.map((file) => file.path);
+          assert.equal(actualPaths.sort(), expected.sort());
+        }
+      } else if (mode === 'watch') {
+        const actual: string[] = [];
```

Comment:
Nit: Could add a symmetrical comment here with a similar sentiment to "`// One-time globbing (via fast-glob).`.

Reply:
Removed the other comment, since it's pretty much self-describing